staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -708,6 +722,7 @@ func (o *DebugOptions) generatePodCopyWithDebugContainer(pod *corev1.Pod) (*core
 			Name:        o.CopyTo,
 			Namespace:   pod.Namespace,
 			Annotations: pod.Annotations,
+			Labels:      pod.Labels,
```

Comment:
This is something unrelated to this change?

Reply:
Unless this change, original labels will be removed from copied and we can’t control keep or not them using --keep-labels flag.